cmd/cli/plugin/login/main.go

```diff
@@ -354,6 +356,8 @@ func createServerWithEndpoint() (server *configv1alpha1.Server, err error) {
 		isVSphereSupervisor, err := wcpauth.IsVSphereSupervisor(endpoint, getDiscoveryHTTPClient())
 		// Fall back to assuming non vSphere supervisor.
 		if err != nil {
+			log.Fatalf("Error creating kubeconfig with tanzu pinniped-auth login plugin: %v", err)
+			return nil, err
```

Comment:
@giri-varma More than the use-after-return in L364, I believe intention is to fall through instead of returning to process the else clause in L375. It is not a merge stopper.
Let's address the issue that is already on main and port the fix back to this branch as a followup.

Thanks @anujc25 for catching this.